HelpSource/Classes/HistoryGui.schelp

```diff
@@ -19,6 +19,99 @@ DEFINITIONLIST::
 ## listview || shows either all code::lineShorts::, or the filtered code::lineShorts::; selecting them in listview makes them come appear in post-doc window.
 ::
 
+first example:
+code::
+(
+h = History.new.lines_([
```

Comment:
couldn't you pass the array directly to `new` instead of using the setter?

Reply:
done